### ISSUE TITLE
Ground truth inconsistencies (ShanghaiTech/Part B)

### ISSUE THREAD

**Issue by Tudor67** [desenzhou/ShanghaiTechDataset#2]
Hi,

I trained a convolutional neural network on **ShanghaiTech (part B)** dataset and I found some **ground truth inconsistencies**:
- **Train set:**
  **_1.) Wrong dot annotation maps (only one image):_** **IMG_108.jpg**: 
![image](https://user-images.githubusercontent.com/12481060/56603914-6d89d880-6609-11e9-88ea-d1bef3ead27e.png)
![image](https://user-images.githubusercontent.com/12481060/56603315-3109ad00-6608-11e9-92ba-7b9c16e1e8ac.png)

  **_2.) Unlabeled data:_**
         - **IMG_202.jpg**
![image](https://user-images.githubusercontent.com/12481060/56604332-5f888780-660a-11e9-8d83-f67f614e8385.png)
        - **IMG_296.jpg**
![image](https://user-images.githubusercontent.com/12481060/56605318-8778ea80-660c-11e9-96f5-71935f097e63.png)
        - **IMG_396.jpg**
![image](https://user-images.githubusercontent.com/12481060/56604676-39171c00-660b-11e9-90d5-1fbc431e41a0.png)

   **_3.) More labels per person:_**
          - **IMG_217.jpg**
![image](https://user-images.githubusercontent.com/12481060/56605990-f7d43b80-660d-11e9-936d-ab8e1c636014.png)

**- Test set:**
      **_1.) Wrong dot annotation maps:_ None.**
     **_2.) Unlabeled data:_ IMG_131.jpg, IMG_175.jpg, IMG_303.jpg, ...**
     **_3.) More labels:_ IMG_185.jpg, IMG_283.jpg, ...**

I would like to know if this is the final version of the dataset (that is used in various papers related to crowd counting). I downloaded the dataset from https://www.dropbox.com/s/fipgjqxl7uj8hd5/ShanghaiTech.zip?dl=0

Thanks.
